### PR TITLE
Add testLoader for validation purposes.

### PR DESCRIPTION
**Pull request by jafyvilla**
Change-Id: I2b4ba99dd764be6ffe911a44c161153f0bc6cb92

### CODE REVIEW

File: fivox/testLoader.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "event.h"
+#include "testLoader.h"
+#include "uriHandler.h"
+
+#ifdef final
```
Comment (tribal-tec) [inline fivox/testLoader.cpp]: wtf?

File: fivox/testLoader.h [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ */
+class TestLoader : public EventSource
+{
+public:
+    /**
+    * Construct a new test event source.
+    *
+    * @param params the URIHandler object containing the parameters
+    * to define the event source
+    * @throw H5::exception or std::exception on error
```
Comment (tribal-tec) [inline fivox/testLoader.h]: liar!

File: fivox/types.h [hunk 3]
```diff
@@ -67,6 +67,7 @@ using vmml::AABBf;
 enum VolumeType
 {
     TYPE_UNKNOWN,      //!< Unknown URI scheme
+    TYPE_TEST,         //!< Dummy type for validation
```
Comment (tribal-tec) [inline fivox/types.h]: maybe called it dummy or validation then?

File: fivox/testLoader.cpp (hunk 1 above)
Comment (eile) [inline fivox/testLoader.cpp]: itk wtf

**Comment by jafyvilla** [comment]
Retest this please

**Comment by jafyvilla** [comment]
Retest this please

File: fivox/types.h (hunk 3 above)
Comment (jafyvilla) [inline fivox/types.h]: I don't like dummy as a name, and validation is too long.

**Comment by jafyvilla** [comment]
Retest this please

**Comment by jafyvilla** [comment]
ping

**Pull request merged**